From 5a8fdb632a0cd37a957e622d84c9f045018fe2e4 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Mon, 4 Aug 2025 08:34:50 +0200
Subject: [PATCH 48/55] yuncore_ax820: add insta1/2 partitions

Adds flash partitions for InstaCert certificate storage on YunCore AX820, supporting dual-bank certificate management.

Signed-off-by: John Crispin <john@phrozen.org>
---
 target/linux/ramips/dts/mt7621_yuncore_ax820.dts | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/target/linux/ramips/dts/mt7621_yuncore_ax820.dts b/target/linux/ramips/dts/mt7621_yuncore_ax820.dts
index b8062ceaeb..b4e0dc7b7c 100644
--- a/target/linux/ramips/dts/mt7621_yuncore_ax820.dts
+++ b/target/linux/ramips/dts/mt7621_yuncore_ax820.dts
@@ -140,7 +140,17 @@
 			partition@90000 {
 				compatible = "denx,uimage";
 				label = "firmware";
-				reg = <0x90000 0xf60000>;
+				reg = <0x90000 0xf40000>;
+			};
+
+			partition@fd0000 {
+				label = "insta1";
+				reg = <0xfd0000 0x10000>;
+			};
+
+			partition@fe0000 {
+				label = "insta2";
+				reg = <0xfe0000 0x10000>;
 			};
 
 			partition@ff0000 {
-- 
2.34.1

